src/components/AppRouter/AppRouter.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import { Routes, Route } from "react-router-dom";
+import { authRoutes, publicRoutes } from "../../routes";
+import { isAuth } from "../../data/Data";
+
+const AppRouter = () => {
+  return (
+    <Container style={{ padding: "40px" }}>
```

Comment:
try to keep styles in the same place (.css or here or check some css libraries for react)

Reply:
fixed